orchagent/intfsorch.cpp

```diff
@@ -67,7 +67,6 @@ void IntfsOrch::doTask(Consumer &consumer)
             Port port;
             if (!gPortsOrch->getPort(alias, port))
             {
-                SWSS_LOG_INFO("Failed to locate interface %s\n", alias.c_str());
```

Comment:
it might happen when we create an interface and the message comes first before the corresponding port/vlan/lag is created. I will later resolve this by introducing dependency relations and ref count for port.